### DISCUSSION TITLE
Frequency of regular coffea releases? One month?

### DISCUSSION THREAD

**Discussion by lgray** [scikit-hep/coffea#458] [General]
Hi Everyone,

Given that things are becoming more steady-state in terms of dependencies and, updates, and bugfixes - it makes sense to have a more paced released schedule than "when it seems like a good time". 

I would propose that we make a coffea release at the end of every month. I think that's a reasonable amount of time to accumulate new features and bugfixes. We could start this at the end of the month, if someone wants a longer or shorter release schedule. Let's discuss it here!

**Comment by nsmith-** [comment]
I would say once a month is the minimum frequency (lest we get no updates) but shouldn't preclude more releases. At least I'd like to keep using [SemVer](https://semver.org/) and not move to [CalVer](https://calver.org/) for now, though it is rather appropriate for this project.

**Reply by lgray** [reply]
Yeah - I'd be happy keeping with SemVer, and indeed once per month at a minimum, not precluding emergency patches and such.

Maybe the best strategy is to have releases for emergency patches and then otherwise keep a weekly rc running till the next month ticks?

Anyway - let's try for version 0.7.3 on May 7th. Keep it to the first Friday of every month?
